### PR TITLE
Zsu678 brighton:sparkling_heart:

### PR DESCRIPTION
**Pull request by zsu678**
what do you think

### CODE REVIEW

File: zsu686.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle:zsu678
```
Comment (impappl) [inline zsu686.json]: Add a space between : and zsu678

File: zsu686.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+latitude:50.822965
```
Comment (impappl) [inline zsu686.json]: Same here!

**Comment by hollenberry** [comment]
Hey @zsu678 ! Almost there! You'll need to rename your file into a _pins/ directory, like @MahmoudElgendy has done. Otherwise, perfect.

If you need any help, please check out the following video: https://vimeo.com/201011288, or reach out again and let us know if you have questions!

**Comment by crichID** [comment]
@zsu678 I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6702/commits/a65523d728e37946190228c5353b8d4c094535cb and https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6702/commits/3d0907bc875c3e5a96c89f836c36de6a57ce62a2. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 26th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**